app/index.ejs

```diff
@@ -1,201 +1,346 @@
+        <div class="col-xs-12">
+          <ul class="list-inline preview-nav">
+            <li>
+              <a ng-href="eventOverview/{{ globalPage.confId }}" translate
+                >Overview</a
+              >
+            </li>
+            <li>
+              <a ng-href="eventRegistrations/{{ globalPage.confId }}" translate
+                >Registrations</a
+              >
+            </li>
+            <li>
+              <a ng-href="eventDetails/{{ globalPage.confId }}" translate
+                >Details</a
+              >
+            </li>
+            <li>
+              <a ng-href="eventForm/{{ globalPage.confId }}" translate
+                >Questions</a
+              >
+            </li>
+          </ul>
         </div>
       </div>
     </div>
-  </section>
-</div>
-
-<div ng-if="globalPage.type === 'registration'">
-  <header id="frontheader">
-  </header>
-  <section id="frontpagetitle">
-    <div class="container">
-      <div class="row spacing-above-md">
-        <div class="col-sm-10 col-xs-12 stacked-spacing-col-sm" ng-if="globalPage.confId">
-          <a ng-href="{{registerMode || 'register'}}/{{globalPage.confId}}/page/">
-            <h1 ng-bind="globalPage.conference.name"></h1>
-          </a>
-          <p class="spacing-above-md"><i class="fa fa-calendar fa-fw"></i>
-            {{globalPage.conference.eventStartTime | evtDateFormat: globalPage.conference.eventTimezone:'ddd, MMM D, YYYY h:mma'}} -
-            {{globalPage.conference.eventEndTime | evtDateFormat: globalPage.conference.eventTimezone:'ddd, MMM D, YYYY h:mma'}}
-            <span ng-if="globalPage.conference.locationName"><br><i class="fa fa-map-marker fa-fw"></i>
-              {{globalPage.conference.locationName}}</span>
-          </p>
-        </div>
-        <div class="col-sm-10 col-xs-12 stacked-spacing-col-sm" ng-if="!globalPage.confId">
-          <a href="">
-            <h1 ng-bind="currentEventName"></h1>
-          </a>
-        </div>
-        <div class="col-sm-2 text-right hidden-xs" ng-if="globalUser() && globalUser().authProviderType !== 'NONE'">
-          <a href="logout" class="btn btn-primary btn-important" translate>Sign out</a>
+
+    <main ng-class="globalPage.mainClass" ng-view></main>
+
+    <footer id="pagefooter" ng-if="globalPage.footer">
+      <div class="container">
+        <div class="row">
+          <div class="col-sm-8">
+            <ul class="list-inline">
+              <li>
+                <a href="https://www.cru.org/us/en/about/privacy.html" translate
+                  >Privacy Policy</a
+                >
+              </li>
+              <li><a href="help/" translate>Help</a></li>
+              <% if(!htmlWebpackPlugin.options.prod) { %>
+              <li>
+                <a ng-href="http://localhost:9000/auth/{{ crsToken }}"
+                  >Local Dev Environment</a
+                >
+              </li>
+              <% } %> <% if(!htmlWebpackPlugin.options.prod) { %>
```

Comment:
I think you could just keep the same `if` going for both links.

Reply:
Yeah you're right, changed it :+1: